frontend/src/component/personalDashboard/Grid.tsx

```diff
@@ -1,6 +1,74 @@
 import { Box, Grid, styled } from '@mui/material';
 import type { Theme } from '@mui/material/styles/createTheme';
 
+export const ContentGridContainer = styled('div')({
+    containerType: 'inline-size',
+});
+
+const ContentGrid2 = styled('article')(({ theme }) => {
+    return {
+        backgroundColor: theme.palette.divider,
+        borderRadius: `${theme.shape.borderRadiusLarge}px`,
+        overflow: 'hidden',
+        border: `0.5px solid ${theme.palette.divider}`,
+        gap: `2px`,
+        display: 'grid',
+
+        '&>*': {
+            backgroundColor: theme.palette.background.paper,
+        },
```

Comment:
make sure all direct children have a background color. This can be pushed to the gird items if we want.